<ui version="4.0" >
 <class>colorFormClass</class>
 <widget class="QDialog" name="colorFormClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>colorSetting</string>
  </property>
  <widget class="QTableWidget" name="tableWidget" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>70</y>
     <width>281</width>
     <height>261</height>
    </rect>
   </property>
   <property name="selectionMode" >
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior" >
    <enum>QAbstractItemView::SelectItems</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Set</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>30</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
